tools/account_diff_table.py

```
@@ -0,0 +1,195 @@
+***6014531444572197=491212058001? 0807   23-04   ok
+"""
+
+from __future__ import annotations
+
+import argparse
+import csv
+import json
+import re
+from collections import defaultdict
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.15.10)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)